templates/vhost/_filters.epp

```diff
@@ -0,0 +1,8 @@
+<% unless $filters.empty {-%>
+
+  ## Filter module rules
+  ## as per http://httpd.apache.org/docs/2.2/mod/mod_filter.html
```

Comment:
Apache 2.2 support was dropped

```suggestion
  ## as per http://httpd.apache.org/docs/2.4/mod/mod_filter.html
```

Reply:
Thanks for the update will resolve in upcoming commits ..

Reply:
Done